### PR TITLE
Implement a custom reddit scraper using a personal account.

### PR DESCRIPTION
**Pull request by Sid-Data-Universe**

### CODE REVIEW

File: scraping/reddit/reddit_custom_scraper.py [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+import traceback
+import datetime as dt
+import asyncio
+import random
+import os
+
+
+class RedditCustomScraper(Scraper):
+    """
+    Scrapes Reddit data using the Reddit Scraper Lite actor.
```
Comment (RusticLuftig) [inline scraping/reddit/reddit_custom_scraper.py:23]: nit: update

File: scraping/reddit/reddit_custom_scraper.py [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+    """
+
+    USER_AGENT = "User-Agent: python:data-universe-app:v1: (by /u/Soft-Strawberry-4067)"
+
+    async def validate(self, entities: List[DataEntity]) -> List[ValidationResult]:
+        """Validate the correctness of a DataEntity by URI."""
+        if not entities:
+            return []
```
Comment (RusticLuftig) [inline scraping/reddit/reddit_custom_scraper.py:31]: It should be a bug for us to try to validate no items.

Should raise an exception or at least return a failed ValidationResult?

**Review by RusticLuftig — CHANGES_REQUESTED** [review]
Overall LGTM. Question about the release though. Since this is going into main, should this change also update the DataDesirabilityLookup to weight Twitter more heavily? Also, can we add some docs? 

Alternatively, should this get merged into a feature branch and then once everything is ready we can merge that into main?

File: scraping/reddit/reddit_custom_scraper.py (hunk 1 above)
Comment (Sid-Data-Universe) [reply scraping/reddit/reddit_custom_scraper.py:23]: Fixed in next push.

File: scraping/reddit/reddit_custom_scraper.py (hunk 2 above)
Comment (Sid-Data-Universe) [reply scraping/reddit/reddit_custom_scraper.py:31]: Consistent across all scrapers. Will resolve in a follow up across them.

Comment (RusticLuftig) [reply scraping/reddit/reddit_custom_scraper.py:36]: SG. ty

**Review by RusticLuftig — APPROVED** [review]

**Pull request closed without merging**